core/scrollbar.js

```diff
@@ -272,6 +284,36 @@ Blockly.Scrollbar.prototype.dispose = function() {
 };
 
 /**
+ * Set the length of the scrollbar's knob and change the SVG attribute
+ * accordingly.
+ * @param {number} newLength The new scrollbar handle length.
+ */
+Blockly.Scrollbar.prototype.setKnobLength = function(newLength) {
```

Comment:
Do you want to mark these as private? or are you planning on changing the way folks call scrollbar from the outside?